ttl/paxinos_rat_ontology.ttl

```diff
@@ -0,0 +1,10095 @@
+### WARNING ###
+### THIS ONTOLOGY IS TEMPORARY DO NOT USE IDENTIFIERS OUTSIDE DEV
+### WARNING ###
+@prefix : <http://www.hbp.FIXME.org/hbp_abam_ontology/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix nsu: <http://www.FIXME.org/nsupper#> .
+@prefix PRBA: <> .
+@base <http://www.hbp.FIXME.org/mba2016_ontology> .
+
+<http://www.hbp.FIXME.org/PRBA_ontology> rdf:type owl:Ontology ;
+                owl:imports <http://www.FIXME.org/nsupper> .
+
+
+PRBA:00000 rdf:type owl:Class ;
+
+                 rdfs:label "root"@en .
```

Comment:
Only last line of the block will ended with a dot. All other block lines have to ended with a semi-colon